crates/bevy_winit/src/lib.rs

```diff
@@ -72,11 +72,53 @@ pub static ANDROID_APP: std::sync::OnceLock<AndroidApp> = std::sync::OnceLock::n
 /// replace the existing [`App`] runner with one that constructs an [event loop](EventLoop) to
 /// receive window and input events from the OS.
 #[derive(Default)]
-pub struct WinitPlugin;
+pub struct WinitPlugin {
+    /// Allows the window (and the event loop) to be created on any thread
+    /// instead of only the main thread.
+    ///
+    /// See [`winit with_any_thread`].
+    ///
+    /// # Supported platforms
+    ///
+    /// Only works on Linux (X11/Wayland) and Windows.
+    /// This field is ignored on other platforms.
+    ///
+    /// [`winit with_any_thread`]: https://docs.rs/winit/latest/winit/event_loop/struct.EventLoopBuilder.html#method.with_any_thread
```

Comment:
```suggestion
    /// [`winit with_any_thread`]: winit::event_loop::EventLoopBuilder::with_any_thread
```

Please avoid linking to docs.rs when the dsymbol in the dependency is available locally.